website/docs/r/vapp_vm.html.markdown

```diff
@@ -282,6 +301,7 @@ example for usage details. **Deprecates**: `network_name`, `ip`, `vapp_network_n
 * `cpu_hot_add_enabled` - (Optional; *v2.10+*) True if the virtual machine supports addition of virtual CPUs while powered on. Default is `false`.
 * `memory_hot_add_enabled` - (Optional; *v2.10+*) True if the virtual machine supports addition of memory while powered on. Default is `false`.
 * `prevent_update_power_off` - (Optional; *v2.10+*) True if the update of resource should fail when virtual machine power off needed. Default is `false`.
+* `sizing_policy_id` (Optional; *v3.0+*, *vCD 10.0+*) VM sizing policy ID. Has to be assigned to Org VDC.
```

Comment:
1. Please explain in this doc "how" to assign to VDC by adding a link to `org_vdc.html#vm_sizing_policy_ids`

2. I have tried setting `sizing_policy_id` to VM **without** assigning to VDC first. The apply showed no errors:
```
~ sizing_policy_id               = "urn:vcloud:vdcComputePolicy:6a23386b-7d0a-40ef-ac4e-db1b9fd7fa04" -> "urn:vcloud:vdcComputePolicy:3ed45350-1951-45ec-8fe5-582bf5ea1336"

<...>

vcd_vapp_vm.demo_vm_wordpress: Still modifying... [id=urn:vcloud:vm:a60a0c5f-0498-4610-8f48-27c7c9de8f5b, 10m0s elapsed]
vcd_vapp_vm.demo_vm_wordpress: Modifications complete after 10m4s [id=urn:vcloud:vm:a60a0c5f-0498-4610-8f48-27c7c9de8f5b]
Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
```

However, the VM didn't get the new policy:

<img width="831" alt="Screenshot 2020-09-21 at 12 56 48" src="https://user-images.githubusercontent.com/11330689/93754143-f4315380-fc09-11ea-8b24-fdc5ac58961e.png">

If update doesn't handle sizing policy:
a. How can we show that to the user?
b. Documentation should state that.

Reply:
for reference, error when not assigned sizing policy is used:
```
Error: [VM creation] error creating VM TestAccVcdVAppEmptyVmWithVmSizingVM : error instantiating a new VM: API Error: 400: [ 00e55772-2d8e-4976-9040-8713c3b5c7f4 ] No organization vDC compute policy found with id: de55a088-2668-44c4-aae4-240353fcc5e5 in vDC: TestAccVcdVdcBasic(com.vmware.vcloud.entity.vdc:02f59c12-69ae-4920-88d4-728a089c1e17).
```

Reply:
I was wrong, we can update on created VM. So updated documentation and implemented.

Reply:
We have seen that it's very troublesome to update (re-create) policy which is already used by a VM and VDC due to how VCD relationships are modeled. You get into reference-abyss where each of the three relates to each other. We discussed with @vbauzysvmware that it looks best to recommend users in the documentation that if they want to change the sizing policy parameters, they should create a new one, apply, and then delete the old one and its references.

CC: @Didainius @dataclouder